_layouts/default.html

```diff
@@ -59,6 +59,13 @@ <h2 class="project-tagline">{{ page.description | default: site.description | de
           </a>
           sandbox project.
         </div>
+        <div>
+          <br>
+          Copyright © 2022 k8gb.io Project Authors. All rights reserved.
```

Comment:
we have `The k8gb Contributors.` in https://github.com/k8gb-io/k8gb/blob/master/.golic.yaml#L19 

Probably better stick with that version for consistency.

Reply:
thank you @ytsarev, amended